http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/service/RuntimeServiceInterp.scala

```diff
@@ -110,6 +110,7 @@ class RuntimeServiceInterp[F[_]: Parallel](config: RuntimeServiceConfig,
                                                       googleProject,
                                                       userInfo,
                                                       petSA,
+                                                      FormattedBy.GCE,
```

Comment:
same question as above

Reply:
oh wait, i see what you're doing after reading the rest of the diff. you're not actually updating the DB 👍

Reply:
yea, this param is called `willBeUsedBy`, so we're checking if `willBeUsedBy` matches are `formattedBy` in DB